barretenberg/cpp/src/barretenberg/vm/generated/avm_prover.hpp

```diff
@@ -16,6 +16,8 @@ class AvmProver {
     using Flavor = AvmFlavor;
     using FF = Flavor::FF;
     using PCS = Flavor::PCS;
+    using Curve = Flavor::Curve;
```

Comment:
Is it normal that there is no diff for avm_prover.cpp at lines 967-968 that shows the removal these type aliases?

Reply:
My bad, missed the cpp changes!